src/app/content/components/Toolbar/MyHighlights/HighlightStyles.tsx

```diff
@@ -0,0 +1,297 @@
+import React from 'react';
```

Comment:
please move all the modal stuff out of `Toolbar` and into `content/highlights/components`